templates/sectionsummary.html

```diff
@@ -37,10 +39,8 @@ <h2>{{summary.title}}</h2>
               {{ summary.empty_list_text }}
             </div>
           {% endif %}
-          {% if summary.add_link %}
-            <div>
-              <a data-qa="add-item-link" href="{{summary.add_link}}">{{summary.add_link_text}}</a>
-            </div>
+          {% if not summary.list.list_items %}
```

Comment:
Could this be an `else` off he initial `if` statement on line 34?

Reply:
This has been removed and no longer applies..